data/management/commands/import_contributors_data.py

```diff
@@ -10,5 +10,6 @@ class Command(BaseCommand):
     IMPORT_DATA = staticmethod(import_data)
 
     def handle(self, *args, **options):
-        for contributor in self.CONTRIBUTORS():
-            self.IMPORT_DATA(contributor)
+        if self.CONTRIBUTORS():
```

Comment:
@sks444 , wouldnt this apply to all of the import commands, if the data is missing / empty?

dont call `self.CONTRIBUTORS()` twice

Reply:
In the OpenHub commands, we won't get the error `TypeError: 'NoneType' object is not iterable` if the data is missing/empty.

Because in the data app we are [returning `None` explicitly](https://github.com/coala/community/blob/master/data/contrib_data.py#L21) when there is any problem in getting the data from the url.

But in the OpenHub commands, we will get the 'empty list' for the same.

Reply:
The point was you are calling `self.CONTRIBUTORS()` twice.

It does work.

Dont waste precious computation time by doing it twice.